src/metabase/models/setting.clj

```diff
@@ -168,7 +168,7 @@
   [:key])
 
 (def ^:private exported-settings
-  '#{application-colors
+  '#{#_application-colors
```

Comment:
Decided to temporarily leave this commented here, so we still have a complete list in the code. This will serve as a useful reference for adding further tests, after which time the entire val will be removed.